docs/user-guide/annotations.md

```diff
@@ -120,26 +119,28 @@ Click on <img src="../img/button/apply_button.jpg" class="ms-docbutton"/> to app
 Managing Annotations
 --------------------
 
-You can manage anytime your annotations through the *Annotation Viewer*. Click on some annotation to display it.
+Once annotations are added to the [TOC](toc.md), the user can **Manage** it by clicking to <img src="../img/button/edit_button.jpg" class="ms-docbutton" /> button from the TOC toolbar and the *Main Annotations panel* will be open.
```

Comment:
maybe we can create a new section "Main Annotations panel" similar to Managing Annotations that can be linked here?